integration/js/utils/WebdriverSauceLabs.js

```diff
@@ -203,6 +203,11 @@ class SaucelabsSession {
       .withCapabilities(cap)
       .forBrowser(capabilities.browserName)
       .build();
+    const timeout = 30000;
+    console.log();
+    let timeouts = await driver.manage().getTimeouts()
+    if(timeouts.script == undefined || timeouts.script < timeout)
```

Comment:
nit: You probably don't need `    const timeout = 30000;` and can just use the value in the if statement directly.

Also there should be a space between `if` and `(` and remove the empty `console.log`

Reply:
Updated, not sure how I ended up with that empty `console.log ` lol